### ISSUE TITLE
Incorrect implementation of advanced activation functions (PReLU, ELU, LeakyReLU) inside Dense layers in CNN architecture

### ISSUE THREAD

**Issue by dhanushsampath18-maker** [Niketkumardheeryan/ML-CaPsule#1431]
While reviewing the CNN model architecture, I found that advanced activation functions are currently being used incorrectly inside Dense layers as string arguments:

```python
model.add(Dense(500, activation='PReLU'))
model.add(Dense(250, activation='ELU'))
model.add(Dense(100, activation='LeakyReLU'))
## Exact Issue:
`PReLU` and `LeakyReLU` are not intended to be passed as string activation arguments inside `Dense()` layers in Keras. These activations should be implemented as separate layers after the Dense layer.

This may cause:
- Runtime or compatibility issues across TensorFlow/Keras versions
- Improper layer behavior
- Reduced maintainability
- Difficulty in upgrading the project to modern frameworks

## Suggested Fix:
Replace:

```python
model.add(Dense(500, activation='PReLU'))
model.add(Dense(500))
model.add(PReLU())
model.add(Dense(250))
model.add(ELU())

model.add(Dense(100))
model.add(LeakyReLU())

Why This Improvement Matters:

For a deep learning medical classification project, proper architecture design is essential for:

Stable model training
Better framework compatibility
Reproducibility
Easier maintenance and future upgrades
Additional Recommended Enhancements:
Add BatchNormalization after Conv2D layers
Improve training stability
Modernize architecture for current TensorFlow/Keras standards
Expected Outcome:
Correct implementation of advanced activations
Better model compatibility
Cleaner architecture
Improved maintainability

GSSoC Contribution:
I would like to work on this issue and submit a PR with the required fixes and improvements.

**Comment by dhanushsampath18-maker** [comment]
I would like to work on this issue under GSSoC. Please assign it to me.

/assign gssoc

**Comment by dhanushsampath18-maker** [comment]
Thanks for assigning this issue to me
